Packs/XDR_iocs/Integrations/XDR_iocs/XDR_iocs.py

```diff
@@ -0,0 +1,360 @@
+import demistomock as demisto
+from CommonServerPython import *
+from CommonServerUserPython import *
+import hashlib
+import secrets
+import string
+import tempfile
+from datetime import timezone
+from typing import Dict, Optional, List, Tuple, Union
+from dateutil.parser import parse
+from urllib3 import disable_warnings
+from math import ceil
+
+
+disable_warnings()
+DEMISTO_TIME_FORMAT: str = '%Y-%m-%dT%H:%M:%SZ'
+
+
+class Client:
```

Comment:
why aren't you using BaseClient?

Reply:
Easy example in Cortex XDR.
also because there is still a chance to URL changes that will be easier in this format.